modules/landing-pages/assets/js/editor/hooks/ui/editor/documents/open/add-landing-pages-tab.js

```diff
@@ -0,0 +1,26 @@
+export class LandingPageAddLibraryTab extends $e.modules.hookUI.After {
+	getCommand() {
+		return 'editor/documents/open';
+	}
+
+	getId() {
+		return 'elementor-landing-pages-add-library-tab';
+	}
+
+	getConditions( args ) {
+		const document = elementor.documents.get( args.id );
+		return 'landing-page' === document.config.type;
+	}
+
+	apply() {
+		$e.components.get( 'library' ).addTab( 'templates/landing-pages', {
+			title: __( 'Landing Pages', 'elementor' ),
+			filter: {
+				source: 'remote',
+				type: 'lp',
```

Comment:
```suggestion
				type: 'landing-page',
```

or explain why `lp`

Reply:
that's the type returned by the Library API